data_safe_haven/README.md

```diff
@@ -47,6 +47,12 @@ Run `dsh deploy shm -h` to see the necessary command line flags and provide them
 > dsh config upload config.yaml
 ```
 
+- As private endpoints for flexible PostgreSQL are still in preview, the following command is currently needed:
+
+```console
+> az feature register --name "enablePrivateEndpoint" --namespace "Microsoft.DBforPostgreSQL"
+```
```

Comment:
Unfortunate, but I don't think it is worth doing this another way if it isn't in Pulumi Azure Native yet.

Reply:
Yes, I think this will be temporary until either:

1. they roll this out more widely
2. direct integration into a VNet is fixed (currently not possible to do this except through the portal)